persistence-cassandra-4.0/pom.xml

```diff
@@ -10,13 +10,13 @@
   <artifactId>persistence-cassandra-4.0</artifactId>
   <properties>
     <!-- If you update this, make sure to keep `cassandra.bundled-driver.version` in sync -->
-    <!-- 4.0.7 depends on 3.11.0: https://mvnrepository.com/artifact/org.apache.cassandra/cassandra-all/4.0.4 -->
+    <!-- 4.0.7 depends on 4.0.0: https://mvnrepository.com/artifact/org.apache.cassandra/cassandra-all/4.0.4 -->
     <cassandra.version>4.0.7</cassandra.version>
     <!--
       The driver used internally by cassandra-all for UDFs (must match the version declared in
       cassandra-all's POM).
     -->
-    <cassandra.bundled-driver.version>3.11.0</cassandra.bundled-driver.version>
+    <cassandra.bundled-driver.version>4.0.0</cassandra.bundled-driver.version>
```

Comment:
And this one as well:

![image](https://user-images.githubusercontent.com/10600041/211510437-4f543705-346e-4419-981c-c37cb64ce23f.png)

@tatu-at-datastax this change must bring a red flag when reviewing.. We already bumped the 4.x to `4.0.7` so either that PR was wrong or this one.. Please, let's not mechanically approve stuff, this has serious implications imo..

Reply:
@ivansenic I did not approve this PR, fwtw, and was suspicious of driver version change. But did not add a strong enough note to wait before merging.